_LowLevelCode/cpp/cpp_communicator/CMakeLists.txt

```diff
@@ -20,12 +20,18 @@ target_include_directories("${MEX_NAME}"
     PRIVATE "${MPI_CXX_INCLUDE_PATH}")
 target_link_libraries("${MEX_NAME}" "${MPI_CXX_LIBRARIES}")
 
-add_subdirectory(serialiser)
 
 if(UNIX)
     # For MPICH on Linux it's necessary to set relative RPATH values so dynamic
     # libraries can be located by the mex library.
-    set(Herbert_EXTERNAL_LIB_REL_PATH "\$ORIGIN/external")
+    message(STATUS "origin: ${ORIGIN}")
+    message(STATUS "MPI root: ${MPI_ROOT}")
+    if (USE_HERBERT_MPI)
+	set(Herbert_EXTERNAL_LIB_REL_PATH "\$ORIGIN/external")
+    else()
+	set(Herbert_EXTERNAL_LIB_REL_PATH "${MPI_ROOT}lib")        
```

Comment:
Missing a "/" unless "/" is in `${MPI_ROOT}`, but that's probably not the best way of handling it.

Reply:
it is in ${MPI_ROOT}. My knowledge of `cmake` is basic.  Do not know how to make it better. (tried)